README.md

```diff
@@ -0,0 +1,13 @@
+# GENIVI Persistence Common Object
+
+This is the official source of the GENIVI Persistence Common Object. At present, all relevant documentation for this project is available in the GENIVI wiki on: https://at.projects.genivi.org/wiki/display/PROJ/The+Persistence+Common+Object
+
+Master location for GENIVI Persistence Common Object: https://github.com/GENIVI/persistence-common-object
+
+## License
+
+For licensing information see the COPYING file, distributed along with this project.
+
+## Build Dependencies and Instructions
+
+The Persistence Common Object has dependencies on the itzam library. The Persistence Common Object specifies an API to access the key/value and configuration data of GENIVI's persistent data handling.
```

Comment:
Itzam is no longer the preferred database, right?
So it should no longer have dependencies on it, I guess?